.scrutinizr.yml

```diff
@@ -0,0 +1,34 @@
+# language: php
+
+filter:
+    paths: [./*]
+    excluded_paths: [./node_modules/*, ./bower_components/*, ./dist]
+
+# before_commands:
+#   - 'composer self-update'
+#   - 'composer update --prefer-stable --prefer-source --no-interaction --no-scripts --no-progress --no-suggest'
+
+tools:
+  php_code_coverage: false
+  php_code_sniffer:
+    enabled: true
+    config:
+      standard: 'PSR2'
```

Comment:
sure. go ahead an add it. do not know if scrutinizr will pick it up though, or if you need that line as well